explorer/src/main/java/com/scalar/client/tool/explorer/Explorer.java

```diff
@@ -92,27 +75,18 @@ private void registerContract(String contractId, String contractClass, String co
       File tmp = File.createTempFile("a_contract", ".class");
       tmp.deleteOnExit();
       Files.copy(inputStream, tmp.toPath(), StandardCopyOption.REPLACE_EXISTING);
-      LedgerServiceResponse response =
-          clientService.registerContract(contractId, contractName, tmp.getPath(), Optional.empty());
-      if (response.getStatus() != StatusCode.OK.get()) {
-        throw new ExplorerException(response.getStatus() + " " + response.getMessage());
-      }
+      clientService.registerContract(contractId, contractName, tmp.getPath(), Optional.empty());
     } catch (IOException e) {
-      throw new ExplorerException(e.getMessage());
+      throw new ExplorerException("Contract not found", StatusCode.CONTRACT_NOT_FOUND);
     }
   }
 
-  public void validate(String assetId) {
-    ClientServiceRequester requester =
-        () -> new ClientServiceResponse(clientService.validateLedger(assetId));
-    fallback(requester);
+  public LedgerValidationResult validate(String assetId) {
+    return clientService.validateLedger(assetId);
   }
 
   public JsonObject listContracts() {
-    ClientServiceRequester requester =
-        () -> new ClientServiceResponse(clientService.listContracts(null));
-    ClientServiceResponse response = fallback(requester);
-    return string2Json(response.getMessage());
+    return clientService.listContracts("");
```

Comment:
```suggestion
    return clientService.listContracts(null);
```

Reply:
@feeblefakie san,
It shows some Ambiguous method call because of two methods (listContracts(String) and listContracts(byte[])) with the same name and different parameters.
So we need to specify which method will be invoked.

Reply:
@scalar-boney Ah, right. Sorry. How about `(String) null` ?
Can you create anther PR for that ?
Also, you can run CI now since most of the workloads are moved to Github Actions.